docs/platforms/react-native/session-replay/privacy/index.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Privacy
+sidebar_order: 5501
+notSupported:
+description: "Learn how to mask parts of your app's data in Session Replay."
+---
+
+<Alert>
+
+Using custom masking in your Session Replays may accidentally expose sensitive customer data. Before publishing an App with Session Replay enabled, make sure to test it thoroughly to ensure that no sensitive data is exposed.
+
+</Alert>
+
+By default, our Session Replay SDK masks all text content, images, webviews, and user input. This helps ensure that no sensitive data is exposed. You can also manually choose which parts of your app's data you want to mask by using the different options listed below.
+
+To disable the default masking behavior (not to be used on applications with sensitive data):
+
+```javascript
+Sentry.mobileReplayIntegration({
+  maskAllText: false,
+  maskAllImages: false,
+  maskAllVectors: false,
+}),
+```
+
+_Make sure your Sentry React Native SDK version is at least 5.36.0 or 6.3.0._
+
+## Mask and Unmask Components
```

Comment:
I: Masking all items doesn't require the mentioned versions, but masking specific items surely do, so I suggest moving this part inside of `Mask and Unmask Components` 
```suggestion
## Mask and Unmask Components

<Note>
Mask and Unmask components requires the Sentry React Native SDK version to be at least 5.36.0 or 6.3.0.
</Note>
```